optima/programs.py

```diff
@@ -531,7 +531,7 @@ def getpars(self, coverage, t=None, parset=None, results=None, ind=0, perturb=Fa
                         if die:
                             raise OptimaException(errormsg)
                         else:
-                            printv(errormsg, 1, verbose)
+                            printv(errormsg, 3, verbose) # WARNING, not sure how serious this is...feels like it shouldn't happen
```

Comment:
@cliffckerr sure, will do. isn't this handled by the applylimits method of parameters though?